File: DAV/tips_scatter_plot.py
import seaborn as sns
import matplotlib.pyplot as plt

# Load the tips dataset
tips = sns.load_dataset("tips")

# Display top 10 records
print("Top 10 Records:")
print(tips.head(10))

# Scatter plot of tip vs day (with jitter)
plt.figure(figsize=(8, 5))
sns.stripplot(x="day", y="tip", data=tips, jitter=True, palette="Set2", edgecolor="gray")
plt.title("Scatter Plot of Tips by Day")
plt.xlabel("Day of the Week")
plt.ylabel("Tip Amount")
plt.grid(True)
plt.show()

File: DAV/tips_scatter_plot.R
# Install and load necessary packages
if (!require(ggplot2)) install.packages("ggplot2")
if (!require(readr)) install.packages("readr")
if (!require(dplyr)) install.packages("dplyr")

library(ggplot2)
library(readr)
library(dplyr)

# Load tips dataset from GitHub
tips <- read_csv("https://raw.githubusercontent.com/mwaskom/seaborn-data/master/tips.csv")

# Display top 10 records
cat("Top 10 Records:\n")
print(head(tips, 10))

# Scatter plot (jittered) of tips vs day
ggplot(tips, aes(x = day, y = tip)) +
  geom_jitter(width = 0.2, height = 0, color = "steelblue", alpha = 0.7) +
  theme_minimal() +
  ggtitle("Scatter Plot of Tips by Day") +
  xlab("Day of the Week") +
  ylab("Tip Amount")
